db/deleteItem_test.go

```diff
@@ -12,7 +12,6 @@ import (
 func TestDeleteItem(t *testing.T) {
 	listID := "474c2Fff7"
 	itemID := "b6cf642d"
-	tableName := "items-table"
```

Comment:
this seems, unrelated to this PR? 😛

Reply:
yeah, I put them in different commits - I didn't want to end up with multiple PRs. Happy to break into different PRs if you want